core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/stopstartus/StopStartUpscaleActions.java

```diff
@@ -128,7 +128,7 @@ protected void doExecute(StopStartUpscaleContext context, StopStartUpscaleGetRec
                         .collect(Collectors.toList());
 
                 List<InstanceMetadataView> stoppedInstancesInHg = instanceMetaDataForHg.stream()
-                        .filter(s -> s.getInstanceStatus() == STOPPED)
+                        .filter(s -> (s.getInstanceStatus() == STOPPED) || (s.getInstanceStatus() == PENDING))
```

Comment:
Same comment as above in StopStartUpscaleHandler ... 1) when will state be set to PENDING, and 2) should this be done during a STOP as well.